extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/indexing/MSQCompactionRunnerTest.java

```diff
@@ -397,39 +398,17 @@ private static CompactionTask.CompactionTuningConfig createTuningConfig(
       PartitionsSpec partitionsSpec
   )
   {
-    return new CompactionTask.CompactionTuningConfig(
-        null,
-        null, // null to compute maxRowsPerSegment automatically
-        null,
-        500000,
-        1000000L,
-        null,
-        null,
-        null,
-        null,
-        partitionsSpec,
-        indexSpec,
-        null,
-        null,
-        !(partitionsSpec instanceof DynamicPartitionsSpec),
-        false,
-        5000L,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null
-    );
+    return TuningConfigBuilder
+        .forCompactionTask()
+        .withMaxRowsInMemory(500000)
+        .withMaxBytesInMemory(1000000L)
+        .withMaxTotalRows(Long.MAX_VALUE)
```

Comment:
Why is this needed?

Reply:
Oh, might have been copied over from somewhere else.
Will get rid of it in a follow up if that's okay (to avoid re-triggering CI.)

Reply:
Sure!